docs/manual/introduction.tex

```diff
@@ -748,6 +750,11 @@
 \item \<-AstubWarnIfRedundantWithBytecode>
   Warn if a stub file entry is redundant with bytecode information; see
   Section~\ref{stub-using}.
+\item \<-AmergeStubsWithSource>
+  If both a stub file and a source file for a class are available, use the
+  greatest lower bound of their annotations. The default behavior (without this
+  flag) is to ignore types from the stub file if source is available. GLB is
```

Comment:
"GLB" is not defined.  Also, even though I know what GLB means, I don't understand this sentence.